atomics/T1566.001/T1566.001.yaml

```diff
@@ -56,3 +56,86 @@ atomic_tests:
     cleanup_command: |
       Remove-Item #{jse_path} -ErrorAction Ignore
     name: powershell
+---
+attack_technique: T1566.001
+display_name: "Phishing: Spearphishing Attachment"
+atomic_tests:
+  - name: RDP Phishing via Outlook Temporary Folders
+    auto_generated_guid: 652d0d34-6565-434f-b069-15dbd824e82c
```

Comment:
`auto_generated_guid` will be automatically generated once the PR is merged as part of our CI/CD pipeline. Can you remove this line as well ?